config.py

```diff
@@ -33,6 +34,10 @@ def __init__(self):
         self.create_xray_when_sending.setChecked(prefs['create_xray_when_sending'])
         self.l.addWidget(self.create_xray_when_sending)
 
+        self.expand_aliases = QCheckBox('Auto associate split aliases')
```

Comment:
I'm not sure this string is particularly self-explanatory..  "Auto-expand aliases" might be a bit clearer, but TBH we likely just need to document somewhere what exactly all this means (forum?  README.md?)